includes/customizer-functions.php

```diff
@@ -0,0 +1,63 @@
+<?php /**
+ * Hello Elementor customizer registration functions.
+ *
+ * @package HelloElementor
+ */
+
+/**
+ * Register Customizer controls which add Elementor deeplinks
+ *
+ * @return void
+ */
+add_action( 'customize_register', 'hello_customizer_register' );
+function hello_customizer_register( $wp_customize ) {
+	// Allow active/inactive via the Experiments
+	if ( ! hello_header_footer_experiment_active() ) return;
+
+	require get_template_directory() . '/includes/controls/customizer-upsell-control.php';
+
+	$wp_customize->add_section(
+		'hello_theme_options',
+		[
+			'title' => __( 'Header &amp; Footer', 'hello-elementor' ),
+			'capability' => 'edit_theme_options',
+		]
+	);
+
+	$wp_customize->add_setting( 'hello-elementor-header-footer', [ 'transport' => 'refresh' ] );
+
+	$wp_customize->add_control(
+		new Hello_Elementor_Upsell(
+			$wp_customize,
+			'hello-elementor-header-footer',
+			[
+				'section' => 'hello_theme_options',
+				'priority' => 20,
+			]
+		)
+	);
+}
+
+
+/**
+ * Enqueue Customiser CSS
+ *
+ * @return string HTML to use in the customizer panel
+ */
+add_action( 'admin_enqueue_scripts', 'hello_admin_print_styles' );
```

Comment:
In all places that enqueue scripts/styles that are exclusive to the header/footer feature, check whether the experiment is registered before these actions or not. If the experiment **is** registered before these enqueue hooks are called, make sure the `add_action` calls are wrapped in an `if` statement checking whether the experiment is active.

Same goes for all `require` statements - if they happen after the experiment is registered - wrap it in a check for `is_feature_active()`

Reply:
Conditioned this out if the experiment is inactive